exporters/etw/include/opentelemetry/exporters/etw/etw_logger.h

```diff
@@ -100,53 +185,38 @@ class Logger : public opentelemetry::logs::Logger
         provHandle(initProvHandle())
   {}
 
-  void Log(opentelemetry::logs::Severity severity,
-           nostd::string_view body,
-           const common::KeyValueIterable &attributes,
-           opentelemetry::trace::TraceId trace_id,
-           opentelemetry::trace::SpanId span_id,
-           opentelemetry::trace::TraceFlags trace_flags,
-           common::SystemTimestamp timestamp) noexcept override
+  nostd::unique_ptr<opentelemetry::logs::LogRecord> CreateLogRecord() noexcept
   {
+    return nostd::unique_ptr<opentelemetry::logs::LogRecord>(new LogRecord());
```

Comment:
This will create a intermediate storage for LogRecord object even when we invoke Logger:Log(). We can keep it as of now, I will try to see it later if we can optimize it.